### PR TITLE
rofi: add page

### PR DESCRIPTION
**Pull request by dikiaap**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/rofi.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+`rofi -show drun`
+
+- Show the list of all commands:
+
+`rofi -show run`
+
+- Switch between windows:
+
+`rofi -show window`
```
Comment (sbrl) [inline pages/linux/rofi.md]: Is there any way to specify which one to switch to? Or is there a graphical interface that shows?

Comment (dikiaap) [reply pages/linux/rofi.md]: Here:
screenshot from 2018-01-14 05-04-17

Comment (sbrl) [reply pages/linux/rofi.md]: Ah, awesome! That looks really useful. In that case, this PR looks ok to me :-)

**Comment by agnivade** [comment]
Thanks for the contribution !

**Pull request merged**